charts/routes/templates/authz-vs.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.authz.enabled }}
+apiVersion: networking.istio.io/v1beta1
+kind: VirtualService
+metadata:
+  name: {{ printf "%s-%s" (include "caraml-routes.fullname" . ) "authz-vs" }}
+  namespace: {{ ternary .Release.Namespace .Values.namespace (ne .Values.namespace "") }}
+  labels:
+    {{- include "caraml-routes.labels" . | nindent 4 }}
+spec:
+  hosts:
+    {{- range .Values.global.hosts.authz }}
```

Comment:
in what scenario do we need to have authz available externally? iirc, we do not expose it

Reply:
you might be right here, I was in a dilemma here too. Although at the moment the service URL added in the mlp values uses the service name to connect to the authz service, I was thinking, would it be better to make it work via the fqdn URL? This would be needed in that case.

Reply:
what doubts me is that calls to AuthZ service do not have AuthN yet

Reply:
Q: When a call is made from say merlin to mlflow, the request would've already passed through AuthN, It would already be an authenticated call, Is my understanding right?

Reply:
sort of, eventually we will add authN using k8s SA for cross-service calls

Reply:
Do you think we shouldn't add this vs until we add the cross-service AuthN?

Reply:
@mbruner wdyt?

Reply:
yeah, I think we should not expose it as of yet

Reply:
removed it for now.